.github/workflows/code_quality.yml

```
@@ -414,39 +414,40 @@ jobs:
         uses: actions/checkout@v4
-      - name: Create required env file
-        run: |
-          echo '${{ secrets.CODE_QUALITY_ENV_FILE }}' > .env
-      - name: Build Django image
-        run: docker compose build
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+      - name: Install Flake8
+        run: pip install $(grep -E '^(flake8|mccabe|pycodestyle|pyflakes)==' src/django/requirements.txt)
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2046:warning:1:13: Quote this to prevent word splitting

(shellcheck)